akka-remote/src/test/scala/akka/remote/serialization/MiscMessageSerializerSpec.scala

```diff
@@ -16,16 +16,15 @@ import java.util.Optional
 import java.io.NotSerializableException
 
 import akka.Done
+import akka.remote.ArteryControlFormats.UniqueAddress
 import akka.remote.routing.RemoteRouterConfig
 import akka.routing._
 
 object MiscMessageSerializerSpec {
   val serializationTestOverrides =
     """
-    akka.actor.serialization-bindings {
-      "akka.remote.serialization.MiscMessageSerializerSpec$TestException" = akka-misc
-      # not enabled by default
-      "akka.Done" = akka-misc
+    akka.actor {
+      serialization-bindings = ${akka.actor.serialization-bindings} { "akka.remote.serialization.MiscMessageSerializerSpec$TestException" = akka-misc } ${akka.actor.java-serialization-disabled-additional-serialization-bindings}
```

Comment:
// reminder how to concat maps

Reply:
not needed, it's merged with reference.conf